templates/dev.rake

```diff
@@ -1,4 +1,4 @@
-if Rails.env.development? || Rails.env.test?
+unless Rails.env.production?
```

Comment:
The main reason we haven't migrated is because we use Bugsnag, which automatically parses environments from `RAILS_ENV`. I see that suspenders is still configured to use Honeybadger.. how do you override this in Honeybadger (we had the same issue there when investigating this split).